src/racer/ast.rs

```diff
@@ -611,6 +611,12 @@ impl<'c, 's> visit::Visitor for ExprTypeVisitor<'c, 's> {
                 });
             }
 
+            ExprKind::Try(ref expr) => {                
+                debug!("try expr");
+                self.visit_expr(&expr);
+                debug!("{:?}", self.result);
```

Comment:
@jwilm, once I know the inner type of `&expr`, how would I convert that to the associated type `Try::Ok`?